### PR TITLE
docs: Update tech docs with Storybook v7

### PR DESCRIPTION
**Pull request by JakePitman**
## Why
For a better docs experience, inline with the new Storybook 7.0 release

## What
Adds autodocs for the following components:

- Brand
- LoadingGraphic
- LoadingHeading
- LoadingInput
- LoadingParagraph
- LoadingSpinner
- Pagination
- GlobalNotification
- InlineNotification

### CODE REVIEW

File: packages/components/src/Brand/_docs/Brand.stories.tsx [hunk 1]
```diff
@@ -10,12 +11,18 @@ export default {
...
+    ],
+    packageName: "@kaizen/Brand",
+    sourceCodeLink:
+      "https://github.com/cultureamp/kaizen-design-system/tree/main/packages/brand",
+    figmaLink: "Add Figma link here",
   },
-}
+} as Meta<typeof Brand>
 
 export const DefaultStory: StoryFn<typeof Brand> = args => <Brand {...args} />
```
Comment (gyfchong) [inline packages/components/src/Brand/_docs/Brand.stories.tsx:27]: We don't need this one anymore, just make a note for what is default in the Variants description.

**Comment by gyfchong** [comment]
Just needs the playground code too: https://github.com/cultureamp/kaizen-design-system/blob/main/packages/button/docs/Button.stories.tsx#L45

File: packages/components/src/Brand/_docs/Brand.stories.tsx (hunk 1 above)
Comment (JakePitman) [reply packages/components/src/Brand/_docs/Brand.stories.tsx:27]: Sweet. This one doesn't have a default as far as I can tell (you always have to pass a `variant` prop). Do I leave it blank in cases like this?

Comment (gyfchong) [reply packages/components/src/Brand/_docs/Brand.stories.tsx]: Yeah, don't mention a default if there is none 😄

File: packages/components/src/Brand/_docs/Brand.stories.tsx [hunk 2]
```diff
@@ -25,42 +48,42 @@ DefaultStory.args = {
...
-        />
-      </div>
-    </StoryWrapper.Row>
-  </StoryWrapper>
+/**
+ * `logo-horizontal`, `logo-vertical`, `enso`, `collective-intelligence`
+ */
+export const Variants: StoryFn = () => (
+  <StickerSheet>
+    <StickerSheet.Row rowTitle="Logo Horizontal">
```
Comment (gyfchong) [inline packages/components/src/Brand/_docs/Brand.stories.tsx:56]: I think make it a single row if possible

Comment (JakePitman) [reply packages/components/src/Brand/_docs/Brand.stories.tsx:56]: Can't seem to get the `'collective-intelligence'` variant to render this way... Even if I give the first three their own row collective intelligence disappears 🤔 
image

Comment (gyfchong) [reply packages/components/src/Brand/_docs/Brand.stories.tsx]: That's super weird... Maybe separate collective intelligence to it's own heading, cause it probably needs a little sentence for why it's used.

**Comment by gyfchong** [comment]
Just having a look through the stories, sorry if I maybe wasn't clear enough at the start (probably should've paired) but you need to split up/make stories for essentially each of the props, similar to [Tooltip](https://dev.cultureamp.design/geoff/KZN-1125-update-docs/?path=/docs/components-tooltip--docs), you also need to create a [separate file for stickersheets](https://github.com/cultureamp/kaizen-design-system/blob/3eca87a72f28f67b114103f50c5f6d67c2569cce/draft-packages/tooltip/docs/Tooltip.stickersheet.stories.tsx) which get placed in a separate `/stickersheets` directory.

Let me know if you want to pair on this a bit

**Comment by gyfchong** [comment]
Sorry, we'll have to do this again later once the packages are re-added to KAIO, the conflicts will be huge.

**Pull request closed without merging**